source/2019-12-14-countdown-to-the-new-year-ember-power-select.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Countdown to The New Year - Ember Power Select
+author: Richard Shea  
+tags: 2019, Addons, DecEmber
+alias: 2019-12-14-countdown-to-the-new-year-ember-power-select
+responsive: true
+---
+
+**This is the fourteenth in our DecEmber series- "Countdown to The New Year: 31 Days of Ember Addons". We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!**
+
+## Day 14
+
+Todays selection for '31 Days of Ember Addons' is [ember-power-select](https://emberobserver.com/addons/ember-power-select)
+
+READMORE
+
+### What It Does
+
+Ember Power Select provides a powerful select component, built from the ground up for Ember so that it works the way you expect an addon to in Ember. Exceptionally [well documented](https://ember-power-select.com/docs), Ember Power Select allows you to create everything from a straightforward list of text items to an asynchronous search driven select which incorporates graphical elements in the list. 
```

Comment:
```suggestion
Ember Power Select provides a powerful select component, built from the ground up for Ember so that it works the way you expect an addon to work in Ember. Exceptionally [well-documented](https://ember-power-select.com/docs), Ember Power Select allows you to create everything from a straightforward list of text items to an asynchronous search driven select which incorporates graphical elements in the list. 
```